tests/test_ui.py

```
@@ -91,51 +79,41 @@ def test_ui_plain_non_tty_skips_prompts(monkeypatch):
         view.input("Enter", default="x")
 
 
-def test_ui_missing_dependencies_raise(monkeypatch):
+def test_ui_choose_interactive_uses_questionary(monkeypatch):
     calls = []
 
-    def fake_which(cmd):
-        calls.append(cmd)
-        return None
+    class DummySelect:
+        def __init__(self, prompt: str, choices: list[str]):
```

Ruff (0.15.9)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)